extra/dhcpcd/build

```diff
@@ -13,4 +13,3 @@ make install
 # Install runit service.
 mkdir -p "$1/etc/sv/dhcpcd"
 cp -f dhcpcd.run "$1/etc/sv/dhcpcd/run"
-ln -sf /run/runit/supervise.dhcpcd "$1/etc/sv/dhcpcd/supervise"
```

Comment:
Does this mean users need to manually symlink both `/run/runit...` and `/var/service...` to enable services using the default (busybox) init?

Reply:
No. The supervise directory is automatically created if it does not exist. This change merely stops supervise from
being a symbolic link to `/run` (this is an optional configuration). So, either supervise will be created by the service manager or the user can setup the symbolic links manually (if they want to do this).

Reply:
excellent